legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/preeval/preeval.pure

```diff
@@ -255,6 +255,22 @@ function <<access.private>> meta::pure::router::preeval::prevalInternal(item : A
   $newItem;
 }
 
+function <<access.private>> meta::pure::router::preeval::preevalAggregateFunction(f:FunctionDefinition<Any>[1], state : meta::pure::router::preeval::State[1], extensions:Extension[*]):PrevalWrapper<Any>[1]

```

Comment:
Should the type of f be a LambdaFunction? They are the only ones that could have open variables.  Probably if it's a concrete function, you want to cleanly create a new set of in scope variables